### PR TITLE
tests - fix web integration tests to fail creating a page

### PR DESCRIPTION
**Pull request by bpasero**

### CODE REVIEW

File: test/integration/browser/src/index.ts [hunk 1]
```diff
@@ -35,11 +35,6 @@ type BrowserType = 'chromium' | 'firefox' | 'webkit';
 async function runTestsInBrowser(browserType: BrowserType, endpoint: url.UrlWithStringQuery, server: cp.ChildProcess): Promise<void> {
 	const browser = await playwright[browserType].launch({ headless: !Boolean(optimist.argv.debug) });
 	const context = await browser.newContext();
-	try {
-		await context.grantPermissions([`clipboard-read`, `clipboard-write`]); // some tests need clipboard access
```
Comment (deepak1556) [inline test/integration/browser/src/index.ts:39]: Is the permission not needed any more ?

Comment (bpasero) [reply test/integration/browser/src/index.ts:39]: @deepak1556 playwright does not seem to support this, I filed https://github.com/microsoft/playwright/issues/19888

Comment (deepak1556) [reply test/integration/browser/src/index.ts:39]: I think with https://chromium-review.googlesource.com/c/chromium/src/+/1895749, there is only `clipboard-write` with `allowWithoutSanitization: <Boolean>` property set. I wonder if playwright needs to update the permission name.

```
clipboard-read-write -> clipboard-write with allowWithoutSanitization: true
clipboard-sanitized-write -> clipboard-write with allowWithoutSanitization: false
```

**Review by deepak1556 — APPROVED** [review]

File: test/integration/browser/src/index.ts (hunk 1 above)
Comment (bpasero) [reply test/integration/browser/src/index.ts:39]: @deepak1556 oh nice findings, maybe tell them in the issue?

**Pull request merged**